src/AddTodoForm.jsx

```diff
@@ -1,9 +1,16 @@
-const AddTodoForm = () => {
+const AddTodoForm = (props) => {
+  const handleAddTodo = (event) => {
+    event.preventDefault();
+    const todoTitle = event.target.title.value;
+    console.log(todoTitle);
+    event.target.reset();
```

Comment:
No need to change now, but just wanted to remind you that it's good practice to reset values at the end of the function

Reply:
@riousenkai Thank you! I will change it with the next PR.